src/Tribe/Assets.php

```diff
@@ -90,6 +90,7 @@ public function register_in_wp( $assets = null ) {
 	 */
 	public function enqueue( $forcibly_enqueue = null ) {
 		$forcibly_enqueue = (array) $forcibly_enqueue;
+		$localized = array();
```

Comment:
Good call and done!
BTW you can use the `feature/80614-ticket-tables-ui` branch of event tickets to test it - I've got it set up in `event-tickets/src/Tribe/Metabox` for testing.